assets/src/styles/main.css

```diff
@@ -2,26 +2,24 @@
 @tailwind components;
 @tailwind utilities;
 
-@layer components {
```

Comment:
Using `@layer` means that Tailwind is removing the styles in this section if they are not found in the template files. As we are pulling the readme content from GitHub, we should always include the styles.